inject/android.js

```diff
@@ -16,10 +30,34 @@ function initCode () {
         platform: processedConfig.platform
     })
 
+    processedConfig.messageSecret = messageSecret
     init(processedConfig)
 
-    // Not supported:
-    // update(message)
+    // @ts-ignore
+    window.clickToLoadMessageCallback = (message) => {
+        update(message)
+    }
+
+    window.addEventListener('sendMessageProxy' + messageSecret, event => {
+        event.stopImmediatePropagation()
+
+        if (!(event instanceof CustomEvent) || !event?.detail) {
+            return console.warn('no details in sendMessage proxy', event)
+        }
+
+        const messageType = event.detail?.messageType
+        if (!allowedMessages.includes(messageType)) {
+            return console.warn('Ignoring invalid sendMessage messageType', messageType)
+        }
+
+        const message = {
+            type: messageType,
+            options: event.detail?.options
+        }
+        const stringifiedArgs = JSON.stringify(message)
+        // @ts-ignore
+        window.ClickToLoadAndroid.process(stringifiedArgs)
```

Comment:
This is exposed through Android right?
I think we could probably make this not CTL specific; even if we're acknowledging that it's legacy upfront. `window.legacyContentScopeScriptsMessageHandler....` etc

Reply:
That makes sense, I've renamed them to `legacyContentScopeScriptsMessageHandler` and `legacyContentScopeScriptsMessageCallback`.